### ISSUE TITLE
4chan.org both captcha 1 / 2 not working

### ISSUE THREAD

**Issue by starfear** [f77/Dashchan#9]
![IMG_20191216_201855](https://user-images.githubusercontent.com/46917875/70899049-98fe7f00-2041-11ea-8171-5209ceb4ff6b.png)

**Comment by vaultdveller** [comment]
Still an issue, could you (dev) please at least tell us will this be fixed or are you gonna concentrate on making other boards work? dashchan has the best ui out of all 4chan browsers and being forced to only lurk using this sucks, lets say i owe you a pint if this gets fixed

**Comment by darlopvil** [comment]
2.16.0 version didn't fix it, indeed it doesn't seem it worked.
![kek](https://user-images.githubusercontent.com/60988792/84574653-211ed080-ada8-11ea-8439-5d319d9587e0.jpg)

**Comment by irigone** [comment]
Dashchan is based, but having to open Firecucks and an image compressor to post is suffering

**Comment by f77** [comment]
@vaultdveller, no, I don't completely support Dashchan, there is no my goal to make all existed extensions to work.
I just make some very little changes for this client, and it's all.
Dashchan support is a defibrillation of a corpse.

**Comment by vaultdveller** [comment]
Welp, if we ever meet i gotta buy you that pint. Last update fixed captcha if i use captcha2 and turn off javascript for it.
Thank you so much
